(DOCSP-34098)(DOCSP-34247): Updated local tutorial and search tutorial. (#228)
mongodb/docs-atlas-cli: source/atlas-cli-deploy-local.txt

-------------------------------

Use the ``atlas deployments`` command to manage a local |service| 
deployment. You can use the following commands for both local and 
cloud |service| deployments. You can use ``--type local`` or 

> ``--type cloud`` to run the command specifically for local or cloud 

Reviewer: reference: https://preview-mongodbcorryroot.gatsbyjs.io/atlas-cli/DOCSP-34098/command/atlas-deployments-setup/
`Type of deployment that you want to create. Valid values are ATLAS or LOCAL.`
```suggestion
``--type atlas`` to run the command specifically for local or cloud 
```

> deployment. You can use the following commands for both local and 
> cloud |service| deployments. You can use ``--type local`` or 
> ``--type cloud`` to run the command specifically for local or cloud 
> atlas deployments.

Reviewer 2: ```suggestion
deployment. You can use the following commands for both local and 
cloud |service| deployments. You can use ``--type local`` or 
``--type atlas`` to run the command for local or cloud 
atlas deployments respectively.
```

      #. Specify how you want to connect to the deployment and press 
         ``Enter``.

   .. step:: Pause a deployment.

      a. Run the following command to pause a deployment:

         .. code-block:: sh

            atlas deployments pause

>       #. Specify the deployment to pause and press ``Enter``.

Reviewer 3: I usually use the :kbd: directive for the Enter key but not sure what the style guide says on this? Applies to other cases on this page as well
```suggestion
      #. Specify the deployment to pause and press :kbd:`Enter`.
```

   .. step:: Start a deployment

      a. Run the following command to start a deployment:

         .. code-block:: sh

            atlas deployments start
      

>       #. Specify the deployment to start and press ``Enter``.

Reviewer 2: ```suggestion
      #. Specify the deployment to start and press :kbd:`Enter`.
```

   .. step:: Return the logs for the deployment.

>       a. Run the following command to start a deployment:

Reviewer 4: ```suggestion
      a. Run the following command to return logs for a deployment:
```

         .. code-block:: sh

            atlas deployments logs
      

>       #. Specify the deployment to return logs for and press ``Enter``.

Reviewer 2: ```suggestion
      #. Specify the deployment to return logs for and press :kbd:`Enter`.
```

            atlas deployments delete

      #. Specify the deployment to delete and press ``Enter``. 
   

>       #. Specify ``y`` and press ``Enter`` to confirm.

Reviewer 2: ```suggestion
      #. Specify ``y`` and press :kbd:`Enter` to confirm.
```
